src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -71,7 +73,7 @@ export const BasicSliderInput = (props) => {
           value={getValue(value)}
           sx={sliderStyle}
           color={color}
-          size="small"
+          variant="small"
```

Comment:
Note: this fix should probably be in a different commit, especially if it fixes a bug from a released version (so it can be visible in the release notes) :slightly_smiling_face:

Reply:
Agreed. Moreover this is not even a fix.